portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,10 +38,23 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(comments);
+    comments.clear();
```

Comment:
Instead of clearing the comments, we can actually just instantiate the list here.

`ArrayList<String> comments = new ArrayList<String>();`